### PR TITLE
translation scrubbing

### PR DESCRIPTION
**Pull request by Mulesoft-Mysti**
I tried to limit the number of changes to:

1. Making product/feature names that aren't translated conform to the glossary
2. Fixing any vague pronouns, removing unnecessary auxiliary verbs, fixing simple typos.
3. Adding bold for UI text and removing bold from non-UI text, which in this doc set meant tinkering with notes. 

I tested the Note changes and they seemed to work. If you see anything wrong there, please flag it.

### CODE REVIEW

File: modules/ROOT/pages/mq-queues.adoc [hunk 1]
```diff
@@ -37,28 +38,28 @@ See also <<fifoqueues,FIFO Queues>>.
...
-** Encryption (Optional) - Set the queue to encrypted. Anypoint MQ uses PBE with MD5 and triple DES to encrypt messages.
-** [[qdlq]]Assign a Dead Letter Queue (Optional) - Set the dead letter queue (DLQ) name. The DLQ is a previously created queue where undeliverable messages are sent to. For more information, see <<Dead Letter Queues>>.
+** *Default TTL* - How long unprocessed messages persist before being deleted. The maximum TTL is 2 weeks.
+** *Default Lock TTL* - If there is a server failure and a message is not acknowledged, specifies the duration that a message would wait before being put back into the queue. A locked message does not block other messages to be read. The lock makes a message unavailable to other applications while locked. The maximum Lock TTL is 12 hours.
+** *Encryption (Optional)* - Set the queue to encrypted. Anypoint MQ uses PBE with MD5 and triple DES to encrypt messages.
+** [*[qdlq]]Assign a Dead Letter Queue (Optional)* - Set the dead letter queue (DLQ) name. The DLQ is a previously created queue where undeliverable messages are sent to. For more information, see <<Dead Letter Queues>>.
```
Comment (george-hoffer) [inline modules/ROOT/pages/mq-queues.adoc]: This won't work -- You put a bold symbol inside an anchor tag and bolded a non UI item.
Please change to: [[qdlq]] \*Assign a Dead Letter Queue* (Optional)
I'm surprised you were able to do a local build on this PR with the broken anchor tag. Antora should have displayed an error. Also fix line 56: should be \*Encryption* (Optional) or \*Encryption* (optional)

File: modules/ROOT/pages/mq-queues.adoc [hunk 2]
```diff
@@ -37,28 +38,28 @@ See also <<fifoqueues,FIFO Queues>>.
...
-** Encryption (Optional) - Set the queue to encrypted. Anypoint MQ uses PBE with MD5 and triple DES to encrypt messages.
-** [[qdlq]]Assign a Dead Letter Queue (Optional) - Set the dead letter queue (DLQ) name. The DLQ is a previously created queue where undeliverable messages are sent to. For more information, see <<Dead Letter Queues>>.
+** *Default TTL* - How long unprocessed messages persist before being deleted. The maximum TTL is 2 weeks.
+** *Default Lock TTL* - If there is a server failure and a message is not acknowledged, specifies the duration that a message would wait before being put back into the queue. A locked message does not block other messages to be read. The lock makes a message unavailable to other applications while locked. The maximum Lock TTL is 12 hours.
+** *Encryption (Optional)* - Set the queue to encrypted. Anypoint MQ uses PBE with MD5 and triple DES to encrypt messages.
```
Comment (george-hoffer) [inline modules/ROOT/pages/mq-queues.adoc]: Please don't bold (Optional). Change to \*Encryption* (Optional).

File: modules/ROOT/pages/mq-apis.adoc [hunk 3]
```diff
@@ -140,12 +140,12 @@ In the Broker API portal, click *Download* to download the API as a RAML, Mule 3
...
 
 [[mqstatapi]]
-== MQ Stats API
+== Anypoint MQ Stats API
 
 * Browse statistics for queues and message exchanges.  
 * View the number of messages currently in a queue. 
 * View historical data for queue activity over time. 
-* View your entire organization's usage of MQ for billing purposes.
+* View your entire organization's usage of Aypoint MQ for billing purposes.
```
Comment (george-hoffer) [inline modules/ROOT/pages/mq-apis.adoc]: Typo -- Aypoint

File: modules/ROOT/pages/mq-access-management.adoc [hunk 4]
```diff
@@ -50,18 +50,18 @@ You can use this display to delete permissions if needed.
...
 You can create a role that you can apply to other users in your organization.
-While you don't need roles to complete this tutorial, when you use MQ as an administrator or developer, you should create
+While you don't need roles to complete this tutorial, when you use Anypoint MQ as an administrator or developer, you should create
 roles for all those in your organization who use MQ.
 
 Creating a role lets you assign access rights to users in your organization, such as for administrators, developers, or for those who only view information but don't change it.
 
-. In Anypoint Platform, click Access Management and Roles.
-. Click Add Role.
+. In Anypoint Platform, click *Access Management and Roles*.
```
Comment (george-hoffer) [inline modules/ROOT/pages/mq-access-management.adoc]: Should be \*Access Management* and \*Roles* or \*Access Management* > \*Roles*

File: modules/ROOT/pages/mq-apis.adoc [hunk 5]
```diff
@@ -621,21 +621,24 @@ The steps in this section are summarized.
...
 You can set up a Studio project with:
 
-* An HTTP connector: Host 0.0.0.0, Port 8081, and path in the properties menu set to the `/mq/{messageId}` value.
-* Anypoint MQ connector with the client ID and secret you got from Anypoint Platform > MQ, and the Destination set to the queue you created in Postman.
-* Logger with Message set to the `#[payload]` value.
+* An HTTP connector: 
+  ** Host `0.0.0.0`
+  ** Port `8081`
+  ** Path in the properties menu set to `/mq/{messageId}`.
+* Anypoint MQ connector with the client ID and client secret from *Anypoint Platform > MQ*, and the *Destination* set to the queue you created in Postman.
```
Comment (george-hoffer) [inline modules/ROOT/pages/mq-apis.adoc]: Casted > as bold.

**Review by george-hoffer — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/mq-queues.adoc (hunk 1 above)
Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/mq-queues.adoc]: Interesting! It printed the whole thing as bold minus the first square bracket: [[qdlq]]Assign a Dead Letter Queue (Optional). fixed.
There's no link to that location in this doc--does the [[anchorID]] work across topics?

**Review by george-hoffer — APPROVED** [review]

**Pull request merged**